Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 ruby '2.2.0'
 
 gem 'rails', '4.2.0'
+gem 'json'
```

Comment:
Oh bugger it, I can't bundle install without it at all :/ Might have to remove it again on another machine.